deploy.js

```diff
@@ -1,14 +1,12 @@
-
 const archiver = require('archiver')
 const async = require('async')
 const AWS = require('aws-sdk')
-const fs = require('fs')
 const nanoArgv = require('nano-argv')
 const path = require('path')
 const stream = require('stream')
 
-const regions = require(path.resolve(__dirname, 'lib', 'lambda.regions.json'))
-const template = require(path.resolve(__dirname, 'lib', 'cfn-template.json'))
+const regions = require('./lib/lambda.regions.json')
```

Comment:
windows uses \ , is this going to cause a problem, removing path.resolve (which does this selection for you based on the OS)?

Reply:
Should work on Windows, this is the standard way of requiring. Though lambda is Linux only

Reply:
The deploy.js code runs on the deploying machine / could definitely be windows - I see your source material re: local requires (I never use this on windows so I was being careful, too busy this week to check it myself).